### PR TITLE
Astro 1204 sentence case

### PR DESCRIPTION
**Pull request by Cheshire89**
https://rocketcom.atlassian.net/browse/ASTRO-1204

Applied sentence case to following attributes

placeholder
label
select list items if applicable

### CODE REVIEW

**Review by dmcalester — APPROVED** [review]

File: stories/rux-form-elements.stories.js [hunk 1]
```diff
@@ -333,7 +333,7 @@ export const SelectMenu = () => html`
       <div class="demo-row">
         <label>Enabled</label>
         <select class="rux-select">
-          <option value="">Select an Option</option>
+          <option value="">Select an option</option>
           <optgroup label="Group One">
```
Comment (amalyah) [inline stories/rux-form-elements.stories.js:337]: This label may have to be sentence cased for consistency. Same with Group Two.

File: stories/rux-form-elements.stories.js [hunk 2]
```diff
@@ -352,7 +352,7 @@ export const SelectMenu = () => html`
       <div class="demo-row">
         <label>Disabled</label>
         <select class="rux-select" disabled>
-          <option value="">Select an Option</option>
+          <option value="">Select an option</option>
           <optgroup label="Group One">
```
Comment (amalyah) [inline stories/rux-form-elements.stories.js:356]: Sentence case the label for consistency. Same for Group Two.

**Review by amalyah — CHANGES_REQUESTED** [review]
- Need to add in sentence case changes to the rest of the components  as well (re: conversation with Duncan)
- Push button in the working  sample needs to have its label updated.
- The All Push Button Variants page needs to be updated as well.
- The All Slider Variants page needs to be updated.

**Review by cskerbo — APPROVED** [review]

File: src/components/rux-monitoring-icon/README.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Pass properties as attributes of the Astro Monitoring Icon custom element:
 ```xml
 <rux-monitoring-icon
  icon="altitude"
- label="Altitude for Satellite X"
+ label="Altitude for satellite x"
```
Comment (amalyah) [inline src/components/rux-monitoring-icon/README.md:46]: In this example, I suspect that Satellite X is the name of the satellite and would need to be capitalized as a proper noun.

File: src/components/rux-monitoring-icon/README.md [hunk 4]
```diff
@@ -188,8 +188,8 @@ To upgrade to Astro 4 any instance of `<rux-status>` used as an "Advanced Status
 <rux-status
   icon="some-icon"
   status="critical"
-  label="Icon Label"
-  sublabel="Sub Lobel"
+  label="Icon label"
+  sublabel="Sub label"
```
Comment (amalyah) [inline src/components/rux-monitoring-icon/README.md:192]: Sub-label

File: src/components/rux-monitoring-icon/README.md [hunk 5]
```diff
@@ -200,8 +200,8 @@ Would become this
 <rux-monitoring-icon
   icon="some-icon"
   status="critical"
-  label="Icon Label"
-  sublabel="Sub Lobel"
+  label="Icon label"
+  sublabel="Sub label"
```
Comment (amalyah) [inline src/components/rux-monitoring-icon/README.md:204]: Sub-label

File: stories/rux-icons-and-symbols.stories.js [hunk 6]
```diff
@@ -261,7 +261,7 @@ export const MonitoringIconSet = () => html`
         <rux-monitoring-icon
           icon="mission"
           label="Mission"
-          sublabel="Sub Label"
+          sublabel="Sub label"
```
Comment (amalyah) [inline stories/rux-icons-and-symbols.stories.js:264]: Sub-label

File: stories/rux-icons-and-symbols.stories.js [hunk 7]
```diff
@@ -270,7 +270,7 @@ export const MonitoringIconSet = () => html`
         <rux-monitoring-icon
           icon="equipment"
           label="Equipment"
-          sublabel="Sub Label"
+          sublabel="Sub label"
```
Comment (amalyah) [inline stories/rux-icons-and-symbols.stories.js:273]: Sub-label

File: stories/rux-icons-and-symbols.stories.js [hunk 8]
```diff
@@ -279,15 +279,15 @@ export const MonitoringIconSet = () => html`
         <rux-monitoring-icon
           icon="processor"
           label="Processor"
-          sublabel="Sub Label"
+          sublabel="Sub label"
```
Comment (amalyah) [inline stories/rux-icons-and-symbols.stories.js:282]: Sub-label

File: stories/rux-icons-and-symbols.stories.js [hunk 9]
```diff
@@ -279,15 +279,15 @@ export const MonitoringIconSet = () => html`
...
           status="normal"
         ></rux-monitoring-icon>
       </li>
       <li>
         <rux-monitoring-icon
           icon="antenna"
           label="Antenna"
-          sublabel="Sub Label"
+          sublabel="Sub label"
```
Comment (amalyah) [inline stories/rux-icons-and-symbols.stories.js:290]: Sub-label

File: stories/rux-icons-and-symbols.stories.js [hunk 10]
```diff
@@ -296,7 +296,7 @@ export const MonitoringIconSet = () => html`
         <rux-monitoring-icon
           icon="antenna-transmit"
           label="NROL"
-          sublabel="Sub Label"
+          sublabel="Sub label"
```
Comment (amalyah) [inline stories/rux-icons-and-symbols.stories.js:299]: Sub-label

File: stories/rux-slider.stories.js [hunk 11]
```diff
@@ -14,10 +14,10 @@ export const Slider = () => {
   const maxKnob = number('Max', 100);
   const stepKnob = number('Step', 10);
   const valKnob = number('Value', 50);
-  const labelKnob = text('Label', 'Slider Label');
-  const axisLabelsKnob = array('Axis Labels', ['min', 'middle', 'max']);
+  const labelKnob = text('Label', 'Slider label');
+  const axisLabelsKnob = array('Axis labels', ['min', 'middle', 'max']);
```
Comment (amalyah) [inline stories/rux-slider.stories.js:18]: 'Min', 'Middle', 'Max'

File: stories/rux-slider.stories.js [hunk 12]
```diff
@@ -51,12 +51,12 @@ Slider.story = {
...
   const axisLabels2 = ['min', 'middle', 'max'];
-  const label2 = 'Slider Label';
+  const label2 = 'Slider label';
 
   const value3 = 42;
 
   const value4 = 50;
-  const label4 = 'Disabled Slider Label ';
+  const label4 = 'Disabled slider label ';
   const axisLabels4 = ['min', 'middle', 'max'];
```
Comment (amalyah) [inline stories/rux-slider.stories.js:60]: 'Min', 'Middle', 'Max'

**Review by amalyah — CHANGES_REQUESTED** [review]
Only small comments.
- 'Min', 'Middle', 'Max' in Sliders need to have capitalized first letters.
- Monitoring Icon has a proper noun that needs to stay capitalized.

Sub label > Sub-label change is a Nice to Have, not required for approval.

**Comment by dmcalester** [comment]
@Cheshire89 I think this can be merged?

**Review by amalyah — CHANGES_REQUESTED** [review]
Only change request: There is another section of the segmented button readme file further down that says Segment in labels that need to be lower cased.

Other things look great!

**Pull request merged**